es-app/src/CollectionSystemManager.cpp

```diff
@@ -199,20 +255,17 @@ void CollectionSystemManager::updateSystemsList()
 		SystemData::sSystemVector.push_back(mCustomCollectionsBundle);
 	}
 
-	// add auto enabled ones
-	addEnabledCollectionsToDisplayedSystems(&mAutoCollectionSystemsData);
-
 	// create views for collections, before reload
-	for(auto sysIt = SystemData::sSystemVector.begin(); sysIt != SystemData::sSystemVector.end(); sysIt++)
+	for (auto sysIt = SystemData::sSystemVector.begin(); sysIt != SystemData::sSystemVector.end(); sysIt++)
 	{
 		if ((*sysIt)->isCollection())
 		{
 			ViewController::get()->getGameListView((*sysIt));
 		}
 	}
 
-	// if we were editing a custom collection, and it's no longer enabled, exit edit mode
-	if(mIsEditingCustom && !mEditingCollectionSystemData->isEnabled)
+	// if we were editing a collection, and it's no longer enabled, exit edit mode
+	if(mEditingCollection != "" && !mEditingCollectionSystem->isEnabled)  // TODO: consider: what happens if we disable favorites, and all others?
```

Comment:
Currently if we disable the "Favorites" system, the "add to favorites" shortcut still holds. We can disable it in ISimpleGameListView.cpp if we want to. I debated about that and ended up choosing this for consistency, but I'm not particularly attached to either approach.

Reply:
Yeah, I gathered as much, I'm not so sure the consistency argument still holds once there are other auto-collections that might be edited as well, but then again, maybe 'favorites' is a natural default to retain. What do you think?

Reply:
I don't have strong feelings one way or the other. I feel users have no concerns and kind of like the accessibility of "Favorites" at the moment, but I have no evidence to suggest whether they'd miss it if it was removed as the default.

As I mentioned, I personally don't have any strong issues with it, but might suggest keeping it for the time being and then reverting at a later stage if that comes up as an issue. What do you think?